src/cache/cache-localstorage.ts

```diff
@@ -13,8 +13,8 @@ export class LocalStorageCache implements ICache {
     try {
       const payload = JSON.parse(json) as T;
       return payload;
+      /* c8 ignore next 3 */
```

Comment:
c8 was marking the catch statement itself as unexecuted, so had to move this up and ignore the 3 lines